Initium-Odp/src/com/universeprojects/miniup/CommonChecks.java

```diff
@@ -448,7 +458,7 @@ public static boolean checkCharacterIsAdminOfHisGroup(CachedEntity character)
 
 	public static boolean checkCharacterIsIncapacitated(CachedEntity character)
 	{
-		if (checkCharacterIsDead(character) || checkCharacterIsUnconscious(character))
+		if (checkCharacterIsDead(character) || checkCharacterIsUnconscious(character) || checkCharacterIsZombie(character))
```

Comment:
Actually I'm not sure this is a good idea.... it might break zombies from being able to do things, like combat.

Reply:
Yea, a cursory check seems to indicate that this would immediately break zombies. I accepted a previous PR with this change so I have reverted this line in particular in master.